docs/kubernetes/AWSIntegration.md

```diff
@@ -104,7 +103,18 @@ sudo systemctl daemon-reload
 sudo systemctl restart kubelet
 ```
 
-@@@TODO: Add section on `ip rule` config; required for inter-node connectivity due to RPF (see https://github.com/projectcalico/calico-kubernetes/issues/52).
+### Node connectivity workaround
+
+As a temporary workaround to issue https://github.com/projectcalico/calico-docker/issues/426, the following manual steps must be run on each node:
+
+```
+mkdir -p /etc/iproute2
```

Comment:
I'm not sure this line is necessary... iproute2/rt_tables were generated in my cluster by the `kube-up`.